README.md

```diff
@@ -1 +1,81 @@
-# ruby-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ruby Capstone Project
+
+- OOP school library 
+
+## Project Description 
+
+In this project, you will create a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with tables structure analogical to your program's class structure.
+
+- Add new book, music album or game.
+- List all created and saved items
+- test all classes
+
```

Comment:
### Project documentation
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

 - A description of the project.
 - A demo of the project features.
 - You should also highlight some interesting piece of code or something you built that you are very proud of.
 - You all should appear in the video and talk about the project.
 - For recording the video you can use the Zoom recording features while in a call with your peers.

Please your project is missing the above requirements, please kindly add them.